DigitalLearningSolutions.Web/Views/LearningContent/Index.cshtml

```diff
@@ -30,6 +30,6 @@
   </div>
 </div>
 
-<div class="nhsuk-grid-row responsive-iframe-wrapper responsive-iframe-wrapper-padding">
+<div class="nhsuk-grid-row responsive-iframe-wrapper @(@Model.Title.Length > 35 ? "responsive-iframe-wrapper-extra-padding" : "responsive-iframe-wrapper-padding")">

```

Comment:
This solution doesn't work, it is possible to generate a title that will have not trigger this extra padding. e.g. the below with 29 characters. I think we should instead add it as an optional parameter bool in the view model constructor (default to false, set to true for this page)
![image](https://user-images.githubusercontent.com/59561751/132489762-1e3c1ac7-15ce-4272-9350-3bdc14cf35e5.png)

Reply:
I've added an optional LongTitle parameter to the view model and set it to true for this page